.storybook/main.js

```diff
@@ -1,10 +1,12 @@
 const { getWebpackConfig } = require('nuxt')
+const createCompiler = require('@storybook/addon-docs/mdx-compiler-plugin');
 
 module.exports = {
-    stories: ['../components/**/*.stories.js'],
+    stories: ['../components/**/*.stories.(js|mdx)'],
     addons: [
         '@storybook/addon-a11y/register',
         '@storybook/addon-knobs/register',
+        '@storybook/addon-docs/preset'
```

Comment:
Did you try:

```
{
  name: '@storybook/addon-docs',
  options: { configureJSX: true },
}
```

Reply:
i will try it

Reply:
same error with this config. Can you explain me what i miss in my configs maybe i will understand ? Thank's :)